cluster-capi/cluster-api-control-plane.yaml

```diff
@@ -6663,7 +6663,7 @@ spec:
       - args:
         - --leader-elect
         - --metrics-bind-addr=localhost:8080
-        - --feature-gates=ClusterTopology=false,KubeadmBootstrapFormatIgnition=false,LazyRestmapper=false
+        - --feature-gates=ClusterTopology=true,KubeadmBootstrapFormatIgnition=false,LazyRestmapper=false
```

Comment:
I am guessing that the ClusterTopology=true activates/deploys the ClusterTopology Controller (https://cluster-api.sigs.k8s.io/developer/architecture/controllers/cluster-topology.html), right?

Reply:
yes, it seems we need it - our v2 package has it and without it our kind provisioning fails